test/fixtures/dialects/sqlite/quoted_identifiers.sql

```diff
@@ -0,0 +1,8 @@
+SELECT `nih`.`userID`
+FROM `flight_notification_item_history` AS `nih`;
+
+SELECT 'nih'.'userID'
+FROM 'flight_notification_item_history' AS 'nih';
```

Comment:
Yeah, that's correct. Single quotes is just for string literals

Reply:
This technically runs in SQLite, however I'd consider it bad practice personally. I only included it because SQLite allows it to run.

Reply:
Iiiiinteresting. If it runs - I think we keep it. Good catch. I assume it not just runs, but actually gets interpreted as expected too?
```suggestion
-- NOTE: Normally single quoted items are interpreted as strings rather than objects - but this does still run on SQLite.
SELECT 'nih'.'userID'
FROM 'flight_notification_item_history' AS 'nih';
```